.github/workflows/mirror-functions.yaml

```diff
@@ -0,0 +1,28 @@
+name: mirror-functions
+
+on:
+  push:
+    branches:
+      - "*"
+  schedule:
+    - cron: "0 18 * * *"
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set up gcloud
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: "275.0.0"
+          service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+      - run: gcloud auth configure-docker
+      - name: Mirror kpt-functions images
+        run: |
+          ./scripts/mirrorImages.sh kpt-functions ["read-yaml"]="read-yaml" ["write-yaml"]="write-yaml" ["gatekeeper-validate"]="policy-controller-validate"
```

Comment:
I would just hardcode these in the script

Reply:
Hardcoded an associative array